src/libraries/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -833,6 +833,15 @@ public async Task Kill_ExitedNonChildProcess_DoesNotThrow(bool killTree)
                         // process still exists, wait some time.
                         await Task.Delay(100);
                     }
+
+                    DateTime now = DateTime.UtcNow;
+                    if (start.Millisecond + Helpers.PassingTestTimeoutMilliseconds <= now.Millisecond)
+                    {
+                        Console.WriteLine("{0} Failed to kill process {1} started at {2}", now, nonChildProcess.Id, start);
```

Comment:
Does Console.WriteLine do anything when inside XUnit? I thought we have to use ITestOutputHelper.

Reply:
it is captured as console output in CI runs. ITestOutputHelper would be my first choice but it is problematic with RemoteExecutor. We would fail to serialize `this` and we would need to depend on static methods only.

```

```